content/webapp/components/Tags/Tags.New.tsx

```diff
@@ -0,0 +1,111 @@
+import NextLink from 'next/link';
+import { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+import { LinkProps } from '@weco/common/model/link-props';
+import { font } from '@weco/common/utils/classnames';
+import PlainList from '@weco/common/views/components/styled/PlainList';
+import Space from '@weco/common/views/components/styled/Space';
+
+export type TagType = {
+  textParts: string[];
+  linkAttributes: LinkProps;
+};
+
+type PartWithSeparatorProps = {
+  $separator: string;
+  $isLast: boolean;
+};
+
+const nbsp = '\\00a0';
+
+const PartWithSeparator = styled.span.attrs({
+  className: font('intr', 5),
+})<PartWithSeparatorProps>`
+  &::after {
+    display: ${props => (props.$isLast ? 'none' : 'inline')};
+
+    /* non-breaking space (\u00A0) keeps characters that would otherwise break (e.g. hyphens) stuck to the preceding text */
+    content: '${nbsp}${props => props.$separator}${nbsp}';
+  }
+`;
+
+const LinkWrapper = styled(Space).attrs<{ $isLast: boolean }>(props => ({
+  ...(!props.$isLast && { $h: { size: 'l', properties: ['margin-right'] } }),
+}))<{ $isLast: boolean }>`
+  display: inline-block;
+  margin-bottom: ${props => (props.$isLast ? '6px' : '10px')};
+`;
+
+const FancyLink = styled.a`
```

Comment:
I have a strong dislike of the prefix `fancy` but this is probably a me-problem

Reply:
QuickWinLink